docs/guidance/map/beneficial_ownership.md

```diff
@@ -0,0 +1,103 @@
+# Beneficial ownership information
+
+This worked example describes how to:
+*   publish separate contracting and beneficial ownership datasets
+*   publish beneficial ownership data in OCDS
+
+The Beneficial Ownership Data Standard (BODS) [defines](http://standard.openownership.org/en/0.2.0/primer/whatisbo.html) “beneficial ownership” as:
+
+> The right to some share of a legal entity’s income or assets (ownership) or the right to direct or influence the entity’s activities (control).
+
+Publishing the beneficial owners of tenderers and suppliers can help users to:
+
+*   Identify collusion or deception
+*   Detect undeclared conflicts of interest
+*   Identify companies with debarred or sanctioned owners
+*   Assess risk
+
+For more information on the uses of beneficial ownership data in the context of procurement, see [Connecting the dots between beneficial ownership and procurement data](https://medium.com/opendatacoop/connecting-the-dots-between-beneficial-ownership-and-procurement-data-improving-processes-and-2fd0df9d9653). 
+
+OCDS encourages the publication of company names, identifiers, and addresses. But, the core OCDS schema doesn’t cover beneficial ownership data.
+
+Some procurement systems collect beneficial ownership data, usually when companies submit bids. This data is true at the time of collection. But, procurement systems don’t track later changes to the ownership and control of suppliers. This limitation is especially challenging for users interested in longstanding contracts, in which there may have been many changes.
+
+Therefore, the preferred approach is to **publish separate contracting and beneficial ownership datasets**:
+
+*   Publish contracting data using OCDS.
+*   Collect beneficial ownership data in a separate beneficial ownership register
+*   Periodically confirm the accuracy of data in the register and record changes as they happen.
+*   Publish beneficial ownership data using BODS, including a history of changes.
+
+So that users can connect contracting data and beneficial ownership data, you should use the same [organization identifiers](https://standard.open-contracting.org/latest/en/schema/identifiers/#organization-ids) in both datasets.
+
+Alternatively, you can **publish beneficial ownership data in OCDS** using the [beneficial owners extension](https://extensions.open-contracting.org/en/extensions/beneficialOwners/master/) if:
+
+*   You cannot implement a separate beneficial ownership register; or
+*   You collect data in both a beneficial ownership register and a procurement system **and** the threshold for disclosure in the procurement system is lower than in the beneficial ownership register.
```

Comment:
Typically, a register only covers one jurisdiction. It makes sense to collect in the procurement system to be able to have BO data for companies incorporated outside that jurisdiction.

For example, Canada will be creating a BO register for federally-registered corporations. This doesn't cover provincially-registered corporations or international corporations. So, it's still useful to collect data via the procurement system.

Reply:
Hmm, that is kind of covered by the first bullet, but we could add a third bullet e.g. "The existing beneficial ownership register doesn't cover all the existing companies that may participate in a contracting process."

Reply:
I think the guidance is a little too single-country ("you") focused, and it doesn't recognize that in practice there will be multiple BO registers that can be referenced.

OCDS data can use common identifiers with BO registers regardless of whether "you" implement the relevant register. For example, if UK and France have registers but Spain does not, then the UK can use identifiers for British and French suppliers, and use the extension for Spanish suppliers. In this example, the UK ("you") can never implement a BO register for Spanish suppliers – it's impossible.

In practice, very few countries have BO registers. For the foreseeable future, It will *always* be the case that a supplier can participate from a jurisdiction without a register. As such, it always makes sense to use the extension to cover such cases.

Reply:
True, so, what if we change "_Alternatively, you can_..." for "_However, you can also_...", and "_if_: " for "_especially if_", and add the third bullet "_The existing beneficial ownership register doesn't cover all the existing companies that may participate in a contracting process. This may include foreign suppliers that may not be in the local company register_"

Reply:
Thanks - done!